### ISSUE TITLE
Departure altitude waypoint not captured

### ISSUE THREAD

**Issue by vkrizan** [flybywiresim/aircraft#7488]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2022-09-09T10:52:10+00:00",
    "ref": "refs/heads/master",
    "sha": "9a216dfe61228d3a6889f7afb82b42808a7836fa",
    "actor": "frankkopp",
    "event_name": "manual",
    "pretty_release_name": "master:9a216dfe"
}
```

### Describe the bug

A departure altitude waypoint(s) are not being captured for a given SID (which has them). Reproduced on VCBI22 SID OLSAR1D which has `500` as a waypoint.

Flight director is sways to left after passing that waypoint (which is the expected turn), but it corrects it back and directs to fly straight on runway heading.

### Expected behavior

Capture the altitude waypoint on departure and move on the the next one (in SID).

### Steps to reproduce

1. Load in at VCBI runway 22
2. Configure INITA, INITB, PERF
3. Set Departure VCBI22 OLSAR1D
4. Depart
5. Observe waypoint capture

### References (optional)

![20220913213049_1](https://user-images.githubusercontent.com/7695766/189994264-f2877ae1-a77d-4521-b1a2-f1c45447f946.jpg)

### Additional info (optional)

Note, that the AP was engaged after the 500 waypoint, all of this was hand-flying up to the point of the screenshot.

* https://github.com/flybywiresim/a32nx/blob/0e825f47d0e3850d961c85c21632e56db4a82c2f/src/fmgc/src/flightplanning/LegsProcedure.ts#L201-L204
* https://github.com/flybywiresim/a32nx/blob/0e825f47d0e3850d961c85c21632e56db4a82c2f/src/fmgc/src/flightplanning/LegsProcedure.ts#L475-L494

### Discord Username (optional)

coolll.sk#0022

**Comment by 2hwk** [comment]
Fixed as of cFMS v1.5, closed.

**Comment by tracernz** [comment]
We’re still pretty weak on altitude terminated legs. Hopefully we can more tightly integrate LNAV and VNAV to fully fix it.
